### DISCUSSION TITLE
Multi-select for deletion of files

### DISCUSSION THREAD

**Discussion by ghillb** [nvim-neo-tree/neo-tree.nvim#472] [Ideas]
It would be nice if there were the possibility to select multiple files and then do the `delete` action on them. (Similar to the multi select for copying and cutting)

**Comment by minusfive** [comment]
Agreed! Also useful for other operations like cut, copy, move, etc.

**Comment by KrappRamiro** [comment]
+1

**Comment by marcelotaparelli** [comment]
+1

**Comment by pynappo** [comment]
Closed by #2037. `<tab>` will toggle node(s) as selected. if there are nodes selected in this way, _visual versions of commands will be invoked (iff they exist) regardless of whether the command was in normal or visual mode. This means that multi-trash/delete, multi-cut/copy should work as expected.
